crates/core/src/logstore/default_logstore.rs

```diff
@@ -45,17 +58,45 @@ impl LogStore for DefaultLogStore {
     async fn write_commit_entry(
         &self,
         version: i64,
-        tmp_commit: &Path,
+        commit_or_bytes: CommitOrBytes,
     ) -> Result<(), TransactionError> {
-        super::write_commit_entry(self.storage.as_ref(), version, tmp_commit).await
+        match commit_or_bytes {
+            CommitOrBytes::LogBytes(log_bytes) => {
+                match self
```

Comment:
this inner match seems unnecessary, just map the error on put_opts(...).await

Reply:
Ah it was some old code, needed when I also did tmp_commit logic here, will remove it :)